server/bin/pbench-index.py

```diff
@@ -105,6 +105,17 @@ def main(options, name):
                 - Raises an exception caught be a new, outer-most, try/except
                   block that does not have a finally clause (as you don't want
                   any code execution in the finally block).
+        4. Re-evaluate the list of tar balls to index after indexing of the
```

Comment:
Looks like there should be a blank line before this section, as there are for the other sections.

Reply:
yeah sure